versions/1/kibana/spec.yml

```diff
@@ -65,3 +65,25 @@
         type: file
         contentMediaType: "application/json"
         pattern: '^.+\.json$'
+    - description: Folder containing rules
```

Comment:
Please create a sample package and add it to the `test/packages`.

Reply:
I added this asset to `test/packages/good`

Reply:
Unless this should be it's own test package